test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/SwaggerGeneratorTests.cs

```diff
@@ -246,6 +246,75 @@ public void GetSwagger_SetsParameterRequired_IfApiParameterIsBoundToPath()
             Assert.Equal(expectedRequired, operation.Parameters.First().Required);
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void GetSwagger_SetsParameterRequired_IfParameterIsPath(bool isRequired)
```

Comment:
We already have a test for this case (see `GetSwagger_SetsParameterRequired_IfApiParameterIsBoundToPath`), which makes this one redundant.

Reply:
Done